src/components/APIChecker.tsx

```diff
@@ -0,0 +1,71 @@
+import { useState } from 'react';
+
+import { Replay } from '@mui/icons-material';
+import { LoadingButton } from '@mui/lab';
+import { Alert, AlertTitle, Box, Stack, Typography } from '@mui/material';
+
+import { API_HOST } from '../config/env';
+import { useAuthTranslation } from '../config/i18n';
+import { axios, useQuery } from '../config/queryClient';
+import { AUTH } from '../langs/constants';
+
+const APIChecker = (): JSX.Element | null => {
+  const { t } = useAuthTranslation();
+  const [noConnection, setNoConnection] = useState(false);
+  const { isSuccess, isLoading, refetch } = useQuery({
+    queryKey: ['apiStatus'],
+    queryFn: () =>
+      axios
+        .get(`${API_HOST}/status`, { withCredentials: false })
+        .then(({ data }) => data)
+        .catch(() => setNoConnection(true)),
+    retry: 1,
+  });
+
+  if (isSuccess) {
+    return (
+      <Stack direction="row" alignItems="center" color="success.light">
+        <svg viewBox="0 0 10 10" width="20px" height="20px">
+          <circle
+            fill="currentColor"
+            style={{ filter: 'drop-shadow(0px 0px 2px currentColor)' }}
+            r="1.3px"
+            cx="5px"
+            cy="4.8px"
+          />
+        </svg>
+        <Typography
+          sx={{ textShadow: 'currentColor 1px 0px 10px' }}
+          variant="caption"
+        >
+          Connected to Graasp backend
+        </Typography>
+      </Stack>
+    );
```

Comment:
I wouldn't show this. It should be trivial that the backend is connected.

Reply:
Okay, i will remove it then.